CONTRIBUTING.md

```diff
@@ -1,108 +1,130 @@
+This document describes how to set up your development environment and contribute changes to the **microsoft/react-native** project. This is a working fork of **facebook/react-native** where changes for supporting MacOS are being staged. 
+> **Note: This repository will be accepting PRs only specific to MacOS support. To contribute to React Native, please see [Contributing to react-native](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md)**
 
-The [Open Source Guides](https://opensource.guide/) website has a collection of resources for individuals, communities, and companies who want to learn how to run and contribute to an open source project. Contributors and people new to open source alike will find the following guides especially useful:
+This document assumes basic working knowledge with Git and related tools. We are providing instructions specific to this project. You can either do this with the command prompt or with a combination of the command prompt and [**SourceTree**](https://www.sourcetreeapp.com/). 
 
-* [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
-* [Building Welcoming Communities](https://opensource.guide/building-community/)
+## Setting up your branch for changes 
 
+### Creating your own fork
 
-### [Code of Conduct](https://code.fb.com/codeofconduct/)
+If you wish to contribute changes back to the **microsoft/react-native** repository, start by creating your own fork of the repository. This is essential. This will keep the number of branches on the main repository to a small count. There are lots of developers in this project and creating lots of branches on the main repository does not scale. In your own fork, you can create as many branches as you like.
 
-As a reminder, all contributors are expected to adhere to the [Code of Conduct](https://code.facebook.com/codeofconduct).
+- Navigate to **[GitHub](https://www.github.com)** with a browser and log in to your GitHub account. For the sake of this document, let's assume your username is **johndoe**.
+- Navigate to the **[microsoft/react-native](https://github.com/microsoft/react-native)** repository in the same browser session.
+- Click on the **Fork** button at the top right corner of the page.
+- Create the fork under your account. Your GitHub profile should now show **react-native** as one of your repositories.
+- Create a folder on your device and clone your fork of the **Microsoft** repository. e.g. `https://github.com/johndoe/react-native.git`. Notice how your GitHub username is in the repository location.
 
-## Ways to Contribute
+```
+> git clone https://github.com/johndoe/react-native.git
```

Comment:
Suggestion: drop the `> ` prefix before command line commands which is Windows-like and make the code block be ` ```bash ` which is the convention in facebook's setting up docs.